apps/champions/lib/champions/config.ex

```diff
@@ -133,4 +132,38 @@ defmodule Champions.Config do
       )
     end)
   end
+
+  @seconds_in_day 86_400
+
+  def import_dungeon_settlement_levels_config() do
+    {:ok, dungeon_settlement_levels_json} =
+      Application.app_dir(:champions, "priv/dungeon_settlement_levels.json")
+      |> File.read()
+
+    supplies =
+      Users.Currencies.get_currency_by_name_and_game!("Supplies", Utils.get_game_id(:champions_of_mirra))
```

Comment:
Should this currency be in the config file too?

Reply:
I will check with @marco-paulucci!